dtu_mlops_project/configs/datamodule/speechcmd.yaml

```diff
@@ -0,0 +1,38 @@
+_target_: dtu_mlops_project.data.speechcmd.SpeechCommandsDataModule
```

Comment:
I'd consider putting the whole `configs/` directory in the project root (one level higher), the `dtu_mlops_project` is meant to contain source code of the project and all the other static files are also outside of this directory

Reply:
moved